tests/integration/test_requirements.py

```diff
@@ -90,6 +92,8 @@ def test_requirements_generates_requirements_from_lockfile_from_a_category(pipen
             {dev_packages[0]}= "=={dev_packages[1]}"
             [test-packages]
```

Comment:
This would actually just be [test] if the category is `test`.  Same for `doc`

Reply:
Updated.